From 93f6cf15ca06a8de52c3179a6d55ce7a396f9757 Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Mon, 10 Jul 2017 17:18:28 +0200
Subject: [PATCH] cgroup-defs.h: add missing include

fix compiletime error with our 4.1.41 derived kernel
at 82f0f4f012a646a735d6b44de77b7c9d0712c714 and
patch-4.1.40-rt48.patch.gz

| CC      arch/arm/kernel/asm-offsets.s
| In file included from /build/krm/oe-core_2.7.3/build/tmp-glibc/work-shared/colibri-imx6/kernel-source/include/linux/sched.h:62:0,
|                  from /build/krm/oe-core_2.7.3/build/tmp-glibc/work-shared/colibri-imx6/kernel-source/arch/arm/kernel/asm-offsets.c:14:
| /build/krm/oe-core_2.7.3/build/tmp-glibc/work-shared/colibri-imx6/kernel-source/include/linux/cgroup-defs.h:128:21: error: field 'destroy_swork' has incomplete type
|   struct swork_event destroy_swork;
|                      ^~~~~~~~~~~~~

Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 include/linux/cgroup-defs.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/cgroup-defs.h b/include/linux/cgroup-defs.h
index 3fc3b8e..2538559 100644
--- a/include/linux/cgroup-defs.h
+++ b/include/linux/cgroup-defs.h
@@ -16,6 +16,7 @@
 #include <linux/percpu-refcount.h>
 #include <linux/percpu-rwsem.h>
 #include <linux/workqueue.h>
+#include <linux/work-simple.h>
 
 #ifdef CONFIG_CGROUPS
 
-- 
2.9.4

